libs/core.lua

```diff
@@ -523,7 +523,12 @@ local function makeCore(config)
   end
 
   function core.installDeps(path)
-    local meta = pkg.query(gfs, path)
+    local meta, err = pkg.query(gfs, path)
+    -- meta will be `nil` if package is malformed
+    if not meta then
+      error(err)
+      return
+    end
```

Comment:
Since you're just throwing the error a simple `local meta = assert(pkg.query(gfs, path))` would have the same effect and be a lot clearer.